docs/playbook.md

```diff
@@ -50,3 +50,13 @@ In this case, it looks like this:
 +ENV TAMAGO_CHECKSUM="6319b1778e93695b62bb63946c5dd28c4d8f3c1ac3c4bf28e49cb967d570dfd5"
 
 Then follow the instructions, above, for updating the test images.
+
+## Make u-root init verbose
```

Comment:
The other entries in this file seem currently not applicable, should we remove them?

Reply:
maybe in another PR?